kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -33,7 +40,32 @@ public void add(Subscriber subscriber) {
 
     @Override
     public void update(Subscriber subscriber) {
```

Comment:
Should this run in a transaction?  Does the caller provide one?  What happens if this did as well?